components/devfile-page/YAML.tsx

```diff
@@ -19,6 +19,7 @@ interface Props {
  */
 const DevPageYAML = ({ devfileYAML }: Props) => (
```

Comment:
can you change the const name to `yaml` We should probably find a better name for these components as well as the ones the in the home-page directory.